Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -2470,20 +2471,26 @@ scheduledRideExistsPopUpConfig state =
   formatDateInHHMM timeUTC = EHC.convertUTCtoISC timeUTC "HH" <> ":" <> EHC.convertUTCtoISC timeUTC "mm"
 
 
-getPickupMarkerActionImageConifg :: Boolean -> Boolean -> JB.ActionImageConfig 
-getPickupMarkerActionImageConifg pickupFeatureEnabled driverWithinPickupThreshold = 
-  if ((any (\stage -> isLocalStageOn stage) [ RideAccepted, ChatWithDriver]) && pickupFeatureEnabled && driverWithinPickupThreshold) then 
+getDestMarkerActionImageConifg :: ST.HomeScreenState -> Boolean -> JB.ActionImageConfig 
+getDestMarkerActionImageConifg state driverWithinPickupThreshold = do
+  let conditionForPickupImage = any (\stage -> isLocalStageOn stage) [ RideAccepted, ChatWithDriver] && state.data.config.feature.enableEditPickupLocation && driverWithinPickupThreshold
+      conditionForDestinationImage = isLocalStageOn RideStarted && state.data.config.feature.enableEditDestination
```

Comment:
Stage can be RideAccepted also

Reply:
When stage is RideAccepted, we need to show edit markers at both pickup and destination.

Reply:
At rideAccepted stage we show route of driver to source and not of source to destination.